pkgs/development/libraries/fftw/default.nix

```diff
@@ -25,6 +25,8 @@ stdenv.mkDerivation {
     ++ optional withDoc "info"; # it's dev-doc only
   outputBin = "dev"; # fftw-wisdom
 
+  nativeBuildInputs = [ gfortran ];
```

Comment:
I wonder if we should make that optional and add another attribute `fftw-fortran` with the option enabled. There are only very few packages that need the Fortran interface (only cp2k right now).

Reply:
I see that it triggers a lot of rebuilds ... :grimacing: I guess having those bindings does not cause any harm (except a slightly larger file), but I can also move this to a new attribute. Should we get more feedback here on such a central component?

Reply:
Yes, let's go with a separate attribute to keep things clean.

Reply:
Is there any reason you would want it without fortran?

Reply:
@SuperSandro2000 I was referring to `fftw`. I think we should have separate attribute for `fftw` with Fortran enabled. CP2K of course needs to build with fortran.

Reply:
Except the rebuilds I don't see any reason. If adding this as an optional attribute, we get the Fortran bindings on top, but they live in the same *.so files, so it would be somehow redundant. Then again, there is `fftwFloat` `fftwLongDouble`. How would we deal with the Fortran bindings then? `fftwLongDoubleFortran`, `fftwFloatFortran` etc?

Reply:
OK, I was initially worried that Fortran dependencies would end up in the `fftw` closure, but that does seem to be the case.
Let's leave it like it is now but we should re-base it onto the staging branch due to the large number of rebuilds.